web/packages/teleterm/src/services/tshd/types.ts

```diff
@@ -20,23 +20,101 @@ import { SortType } from 'design/DataTable/types';
-export * from 'gen-proto-ts/teleport/lib/teleterm/v1/server_pb';
-export * from 'gen-proto-ts/teleport/lib/teleterm/v1/kube_pb';
-export * from 'gen-proto-ts/teleport/lib/teleterm/v1/app_pb';
-export * from 'gen-proto-ts/teleport/lib/teleterm/v1/label_pb';
+/*
+ *
+ * Do not add new imports to this file, we're trying to get rid of types.ts files.
+ *
+ */
+
+export {
+  /**
+   * @deprecated Import directly from gen-proto-ts instead.
+   */
+  Cluster,
+  /**
+   * @deprecated Import directly from gen-proto-ts instead.
+   */
+  LoggedInUser,
+  /**
+   * @deprecated Import directly from gen-proto-ts instead.
+   */
+  LoggedInUser_UserType,
+} from 'gen-proto-ts/teleport/lib/teleterm/v1/cluster_pb';
+export {
+  /**
+   * @deprecated Import directly from gen-proto-ts instead.
+   */
+  Database,
+} from 'gen-proto-ts/teleport/lib/teleterm/v1/database_pb';
+export {
+  /**
+   * @deprecated Import directly from gen-proto-ts instead.
+   */
+  Gateway,
+  /**
+   * @deprecated Import directly from gen-proto-ts instead.
+   */
+  GatewayCLICommand,
+} from 'gen-proto-ts/teleport/lib/teleterm/v1/gateway_pb';
+export {
+  /**
+   * @deprecated Import directly from gen-proto-ts instead.
+   */
+  Server,
+} from 'gen-proto-ts/teleport/lib/teleterm/v1/server_pb';
+export {
+  /**
+   * @deprecated Import directly from gen-proto-ts instead.
+   */
+  Kube,
+} from 'gen-proto-ts/teleport/lib/teleterm/v1/kube_pb';
+export {
+  /**
+   * @deprecated Import directly from gen-proto-ts instead.
+   */
+  App,
+} from 'gen-proto-ts/teleport/lib/teleterm/v1/app_pb';
+export {
+  /**
+   * @deprecated Import directly from gen-proto-ts instead.
+   */
+  Label,
+} from 'gen-proto-ts/teleport/lib/teleterm/v1/label_pb';
+export {
+  /**
+   * @deprecated Import directly from gen-proto-ts instead.
+   */
+  AuthSettings,
+  /**
+   * @deprecated Import directly from gen-proto-ts instead.
+   */
+  AuthProvider,
+} from 'gen-proto-ts/teleport/lib/teleterm/v1/auth_settings_pb';
+export {
+  /**
+   * @deprecated Import directly from gen-proto-ts instead.
+   */
+  AccessRequest,
+} from 'gen-proto-ts/teleport/lib/teleterm/v1/access_request_pb';
+export {
+  /**
+   * @deprecated Import directly from gen-proto-ts instead.
+   */
+  AccessList,
+} from 'gen-proto-ts/teleport/accesslist/v1/accesslist_pb';
+
+// There's too many re-exports from this file to list them individually.
+// A @deprecated annotation like this Unfortunately has no effect on the language server.
+/**
+ * @deprecated Import directly from gen-proto-ts instead.
+ */
 export * from 'gen-proto-ts/teleport/lib/teleterm/v1/service_pb';
-export * from 'gen-proto-ts/teleport/lib/teleterm/v1/auth_settings_pb';
-export * from 'gen-proto-ts/teleport/lib/teleterm/v1/access_request_pb';
-export * from 'gen-proto-ts/teleport/lib/teleterm/v1/usage_events_pb';
-export * from 'gen-proto-ts/teleport/accesslist/v1/accesslist_pb';
 
 export type {
+  /**
+   * @deprecated Import directly from tshd instead.
+   */
```

Comment:
From what I see, you migrated all places to import `CloneableAbortSignal` from `services/tshd`. We can get rid of  this one.